packages/message-box/__tests__/message-box.spec.ts

```diff
@@ -0,0 +1,183 @@
+import MessageBox from '../src/messageBox'
+import { sleep } from '@element-plus/test-utils'
+
+const selector = '.el-message-box__wrapper'
+
+describe('MessageBox', () => {
+
+  afterEach(() => {
+    const el = document.querySelector('.el-message-box__wrapper')
+    if (!el) return
+    if (el.parentNode) {
+      el.parentNode.removeChild(el)
+    }
+    MessageBox.close()
+  })
+
+  test('create and close', async () => {
+    MessageBox({
+      type: 'success',
+      title: '消息',
+      message: '这是一段内容',
+    })
+    const msgbox: HTMLElement = document.querySelector(selector)
+    expect(msgbox).toBeDefined()
+    await sleep(250)
+    expect(msgbox.querySelector('.el-message-box__title span').textContent).toEqual('消息')
+    expect(msgbox.querySelector('.el-message-box__message').querySelector('p').textContent).toEqual('这是一段内容')
+    MessageBox.close()
+    await sleep(250)
+    expect(msgbox.style.display).toEqual('none')
+  })
+
+  test('invoke with strings', () => {
+    MessageBox('消息', '这是一段内容')
+    const msgbox = document.querySelector(selector)
+    expect(msgbox).toBeDefined()
+  })
+
+  test('custom icon', async () => {
+    MessageBox({
+      type: 'warning',
+      iconClass: 'el-icon-question',
+      message: '这是一段内容',
+    })
+    await sleep(250)
```

Comment:
Please use something more meaningful like `nextTick`

Reply:
will optimize it later